trunk/source/TraceAbstraction/src/de/uni_freiburg/informatik/ultimate/plugins/generator/traceabstraction/AbstractCegarLoop.java

```diff
@@ -287,8 +287,14 @@ public IcfgProgramExecution getRcfgProgramExecution() {
 	public String errorLocs() {
 		return mErrorLocs.toString();
 	}
-
+	
 	public final Result iterate() {

```

Comment:
This is not beautiful.
I want to send the result of the Cegar loop to the client instantly after the loop is done.

- Originally, I did this in TraceAbstractionStarter (after the call of iterate)
- now its here - in a second method, because the original method has so many exit points.

Reason its not in TraceAbstractionStarter anymore:
I got rid of the interactive interface there, less footprint.

Alternative possibility:
introduce a `final Result result` in the original iterate method.

if anyone has a better suggestion, or a clear preerence for another solution please tell me.